seed/data_importer/meters_parser.py

```diff
@@ -297,16 +345,22 @@ def _parse_cost_meter_reading(self, details, meter_details, start_time, end_time
             'conversion_factor': 1
         }
 
-        meter_identifier = '-'.join([str(v) for k, v in meter_details.items()])
+        # Associate this meter reading to each meter and property_id combination
```

Comment:
as stated in another comment:
> This was affected by upstream changes (property_id to property_ids). Specifically, the code is modified to run through _all_ property_ids, instead of just _one_ property_id.
> 
> The changes here are mostly just reordering and indentation.